scripts/nav-tree-utils.js

```diff
@@ -0,0 +1,84 @@
+import { createOptimizedPicture } from './lib-franklin.js';
```

Comment:
Please consider moving this file under `blocks/header` if that's the only place its used.